terminal.go

```diff
@@ -22,5 +23,10 @@ func colorEnabledWriter(w io.Writer) (io.Writer, bool) {
 	if !isatty.IsTerminal(f.Fd()) {
 		return w, false
 	}
+	// Check the TERM environment variable specifically
+	// to check for "dumb" terminals.
+	if os.Getenv("TERM") == "dumb" {
```

Comment:
What about checking $TERM before IsTerminal? It's probably a cheaper.

Also what about extracting the checks so that you have a UseColourOutput predicate or something?